src/state/disconnect-toast-mixin.ts

```diff
@@ -1,37 +1,30 @@
 import { Constructor, LitElement } from "lit-element";
 import { HassBaseEl } from "./hass-base-mixin";
-import { HaToast } from "../components/ha-toast";
-import { computeRTL } from "../common/util/compute_rtl";
+import { fireEvent } from "../common/dom/fire_event";
 
 export default (superClass: Constructor<LitElement & HassBaseEl>) =>
   class extends superClass {
-    private _discToast?: HaToast;
-
     protected firstUpdated(changedProps) {
       super.firstUpdated(changedProps);
       // Need to load in advance because when disconnected, can't dynamically load code.
-      import(/* webpackChunkName: "ha-toast" */ "../components/ha-toast");
+      import(/* webpackChunkName: "notification-manager" */ "../managers/notification-manager");
     }
 
     protected hassReconnected() {
       super.hassReconnected();
-      if (this._discToast) {
-        this._discToast.opened = false;
-      }
+
+      fireEvent(this, "hass-notification", {
```

Comment:
I know these comments are a bit out of scope as we might move to a banner, but…

we should make a helper function for this, just like we do for showing other dialogs.